gtda/utils/tests/test_validation.py

```diff
@@ -1,16 +1,15 @@
-"""Tests for validation functions"""
+"""Tests for validation functions."""
 # License: GNU AGPLv3
 
 import numpy as np
 import pytest
 
-from gtda.utils.validation import (check_diagram, validate_metric_params,
-                                   validate_params)
+from gtda.utils.validation import check_diagram, validate_params
 
 
 # Testing for validate_params
 def test_validate_params():
-    references = {'par1': [int, [0, 1]]}
+    references = {'par1': {'type': int, 'in': [0, 1]}}
```

Comment:
Could you add a comment why these tests fail? (one line foreach)

Reply:
Thanks for suggesting. Done!